src/steps/analyze-addon.ts

```diff
@@ -11,21 +11,24 @@ function getProjectRootDevDependencies(
   };
 }
 
-function getPublicAssets(options: Options): string[] {
-  const { projectRoot } = options;
+function getPublicAssets(options: Options): Record<string, string> {
+  const { packages, projectRoot } = options;
 
   const filePaths = findFiles('public/**/*', {
     projectRoot,
   });
 
-  return filePaths
-    .map((filePath) => {
-      return renamePathByDirectory(filePath, {
-        from: 'public',
-        to: '',
-      });
-    })
-    .sort();
```

Comment:
Since `findFiles()` sorts the file paths already, it's not necessary to call `sort()` again on files that _all_ live in the `public` folder.